Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py

```diff
@@ -3810,48 +3832,23 @@ def upload_file_through_window(step_data):
         return CommonUtil.Exception_Handler(sys.exc_info(), None, "Error parsing dataset")
 
     try:
-        if platform.system() == "Darwin":
```

Comment:
Why this function is reverted? please fix this

Reply:
fixed